taxi/src/main/java/com/epam/training/taxi/Main.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.training.taxi;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+
+        if(args.length == 0)
```

Comment:
As I see, there are two expected arguments, but that check expects at least only one.
What will happen if only one argument was provided to the application? Will we get the expected error message/exception?